// @expo-google-fonts/lato/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Lato_100Thin = require('./100Thin/Lato_100Thin.ttf');
export const Lato_100Thin_Italic = require('./100Thin_Italic/Lato_100Thin_Italic.ttf');
export const Lato_300Light = require('./300Light/Lato_300Light.ttf');
export const Lato_300Light_Italic = require('./300Light_Italic/Lato_300Light_Italic.ttf');
export const Lato_400Regular = require('./400Regular/Lato_400Regular.ttf');
export const Lato_400Regular_Italic = require('./400Regular_Italic/Lato_400Regular_Italic.ttf');
export const Lato_700Bold = require('./700Bold/Lato_700Bold.ttf');
export const Lato_700Bold_Italic = require('./700Bold_Italic/Lato_700Bold_Italic.ttf');
export const Lato_900Black = require('./900Black/Lato_900Black.ttf');
export const Lato_900Black_Italic = require('./900Black_Italic/Lato_900Black_Italic.ttf');
