// @expo-google-fonts/merriweather/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Merriweather_300Light = require('./300Light/Merriweather_300Light.ttf');
export const Merriweather_400Regular = require('./400Regular/Merriweather_400Regular.ttf');
export const Merriweather_500Medium = require('./500Medium/Merriweather_500Medium.ttf');
export const Merriweather_600SemiBold = require('./600SemiBold/Merriweather_600SemiBold.ttf');
export const Merriweather_700Bold = require('./700Bold/Merriweather_700Bold.ttf');
export const Merriweather_800ExtraBold = require('./800ExtraBold/Merriweather_800ExtraBold.ttf');
export const Merriweather_900Black = require('./900Black/Merriweather_900Black.ttf');
export const Merriweather_300Light_Italic = require('./300Light_Italic/Merriweather_300Light_Italic.ttf');
export const Merriweather_400Regular_Italic = require('./400Regular_Italic/Merriweather_400Regular_Italic.ttf');
export const Merriweather_500Medium_Italic = require('./500Medium_Italic/Merriweather_500Medium_Italic.ttf');
export const Merriweather_600SemiBold_Italic = require('./600SemiBold_Italic/Merriweather_600SemiBold_Italic.ttf');
export const Merriweather_700Bold_Italic = require('./700Bold_Italic/Merriweather_700Bold_Italic.ttf');
export const Merriweather_800ExtraBold_Italic = require('./800ExtraBold_Italic/Merriweather_800ExtraBold_Italic.ttf');
export const Merriweather_900Black_Italic = require('./900Black_Italic/Merriweather_900Black_Italic.ttf');
